src/verilogAST.cpp

```diff
@@ -464,4 +464,13 @@ std::string SingleLineComment::toString() {
   return result + "// " + value;
 }
 
+std::string InlineVerilog::toString() {
+    std::string result = value;
+    for (auto &&it : this->interpolated_symbols) {
+          result = std::regex_replace(
+              result, std::regex("\\{" + it.first + "\\}"), it.second->toString());
```

Comment:
Think we should use `{{ }}` for templating since `{ }` could definitely appear for verilog concat. Even `{{ }}` could appear as verilog but think it's significantly less likely.

Also ideally, a templating engine would be used instead of regex. http://goog-ctemplate.sourceforge.net/ is one good example.